### PR TITLE
Add text certification level sections to README

### PR DESCRIPTION
**Pull request by izgeri**
Update the README so that it includes text descriptions of the 
certification levels in addition to the badges, so that it is clearer
in the project READMEs what each certification level is and what it
means

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,28 @@ TODO:
...
-- ![](https://img.shields.io/badge/Certification%20Level-Trusted-007BFF?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
-- ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+## Certification level
+TODO: Select the appropriate certification level section below, and remove all others.
+
+{Community}
+![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+
+This repo is a **Community** level project. It's a community contributed project that **is not reviewed or supported
+by CyberArk engineering**. For more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
```
Comment (brikelly) [inline README.md]: I would drop the "engineering" word from this.

Comment (brikelly) [reply README.md]: (Because we support things as a company, not as a team)

Comment (izgeri) [inline README.md]:
```suggestion
by CyberArk**. For more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
```

**Review by brikelly — APPROVED** [review]
LGTM

**Pull request merged**